contracts/pcv/utils/NamedStaticPCVDepositWrapper.sol

```diff
@@ -88,9 +88,11 @@ contract NamedStaticPCVDepositWrapper is IPCVDepositBalances, CoreRef {
 
         uint256 oldBalance = balance;
         uint256 oldFEIBalance = feiReportBalance;
+        uint256 newBalance = balance - updatePCVDeposit.usdAmount + usdAmount;
+        uint256 newFeiReportBalance = feiReportBalance - updatePCVDeposit.feiAmount + feiAmount;
```

Comment:
Use oldBalance and oldFeiBalance to save another sload

Reply:
added